### ISSUE TITLE
Search for vulnerabilities

### ISSUE THREAD

**Issue by ngrundmann** [hcrudolph/ciphersuite.info#25]

**Comment by hcrudolph** [comment]
Django doesn't seem to find the vulnerabilities' names using the following filter call. However, this does work when executed from the command line: 
```
results_cs = CipherSuite.objects.filter(
	Q(name__icontains=search_term)|
	Q(auth_algorithm__long_name__icontains=search_term)|
	Q(enc_algorithm__long_name__icontains=search_term)|
	Q(kex_algorithm__long_name__icontains=search_term)|
	Q(hash_algorithm__long_name__icontains=search_term)|
	Q(protocol_version__long_name__icontains=search_term)|
	Q(auth_algorithm__vulnerabilities__name__icontains=search_term)|
	Q(enc_algorithm__vulnerabilities__name__icontains=search_term)|
	Q(kex_algorithm__vulnerabilities__name__icontains=search_term)|
	Q(hash_algorithm__vulnerabilities__name__icontains=search_term)
)
```

**Comment by hcrudolph** [comment]
fixed by ea907c5cda8bba8a5b17375cd1c6eda86fd4fb91
